docs/Telemetry.md

```diff
@@ -0,0 +1,53 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+
+## Telemetry
+
+Accessibility Insights for Windows uses telemetry to better understand what features are most helpful to users, as well as to help identify potential issues that users are experiencing.
+
+### Reporting a telemetry action
+Telemetry actions typically indicate a user-initiated action (starting the app, clicking a button, etc.). They are also used to report the result of a user action (for examplle, a user attempt to open a file failed because the file was in an unrecognized format). At this time, telemetry actions are *not* supported in extensions. The telemetry classes are defined in the `AccessibilityInsights.Desktop.Telemetry` namespace. The `Logger` class provides 2 variants of the static `Logger.PublishTelemetryEvent` method:
```

Comment:
>starting the app, clicking a button, etc. [](start = 62, length = 41)

These actions make sense for AIWin, but AxeWindows has no app or button. Should we use samples that are pertinent to axe-windows?